dracut/module-setup.sh

```diff
@@ -72,13 +72,14 @@ install() {
     # timeout script for errors reporting
     inst_hook initqueue/timeout 50 "$moddir/anaconda-error-reporting.sh"
     # python deps for parse-kickstart. DOUBLE WOOOO
+    set -o pipefail
     PYTHONHASHSEED=42 "$moddir/python-deps" "$moddir/parse-kickstart" "$moddir/driver_updates.py" | while read -r dep; do
         case "$dep" in
             *.so) inst_library "$dep" ;;
             *.py) inst_simple "$dep" ;;
             *) inst "$dep" ;;
         esac
-    done
+    done || exit 1
```

Comment:
Note: Instead of `|| exit 1` I'd rather run the script with `bash -eu` -- would that be acceptable?

Reply:
I believe that should be fine. Please make the change, I'll re-run the kickstart tests, where this will run at some point before the boot.iso is done.

Reply:
Done.